libs/platform/src/lib/components/table/table.component.ts

```diff
@@ -426,6 +428,11 @@ export class TableComponent<T = any> extends Table implements AfterViewInit, OnD
     /** @hidden */
     _scrollBarWidth = 0;
 
+    /** @hidden */
+    get _isShownNavigationColumn(): boolean {
+        return this._tableRows.some(row => row.navigatable);
```

Comment:
Iteration in the getter potentially might be a problem.
Let's keep it as standalone value (e.g. observable and async in the template)?